hooks/build

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh -e
+
+WORKDIR="$(pwd)"
```

Comment:
I'd use

```sh
WORKDIR=$PWD
```
directly, no need to call the `pwd` command. (Also, no quotes needed at variable assignments even if the contents contain whitespaces.)

Reply:
makes sense ;-)